### PR TITLE
Add App Version to the helm ls command.

### PR DESCRIPTION
**Pull request by derekbassett**
This will add the feature requested #3592

### CODE REVIEW

**Comment by derekbassett** [comment]
bjornmagnusson something about the rebase did not work correctly.  I have cherry picked and redid.  Looking forward to this being added or some comment about why this won't be added.

**Review by bacongobbler — APPROVED** [review]
Looks fine to me! I'm not sure whether or not this is considered a breaking change, so I'll let the other maintainers chime in here.

**Review by michelleN — APPROVED** [review]
Looks like it is adding functionality not removing, so I don't think it's a breaking change. @bacongobbler 


thanks @derekbassett

**Pull request merged**
